documentation/navigation/multiplatform.md

```diff
@@ -0,0 +1,211 @@
+                }
+            }
+        }
+    }
+}
+```
+
+### Desktop
+
+```kotlin
+fun main() = application {
+    val events: Channel<Events> = Channel()
+    val windowState = rememberWindowState(size = DpSize(480.dp, 640.dp))
+    val eventScope = remember { CoroutineScope(SupervisorJob() + Dispatchers.Main) }
+    
+    Window(
+        state = windowState,
+        onCloseRequest = ::exitApplication,
+        onKeyEvent = {
+            // See back handling section in the docs below!    
+            onKeyEvent(it, events, eventScope) 
+        },
+    ) {
+        YourAppTheme {
+            Surface(color = MaterialTheme.colorScheme.background) {
+                DesktopNodeHost(
+                    windowState = windowState,
+                    onBackPressedEvents = events.receiveAsFlow().mapNotNull {
+                        if (it is Events.OnBackPressed) Unit else null
+                    }
+                ) { 
+                    RootNode(buildContext = it)
+                }
+            }
+        }
+    }
+}
+```
+
+### Web
+
+```kotlin
+fun main() {
+    val events: Channel<Unit> = Channel()
+    onWasmReady {
+        BrowserViewportWindow("Your app") {
+            val requester = remember { FocusRequester() }
+            var hasFocus by remember { mutableStateOf(false) }
+            var screenSize by remember { mutableStateOf(ScreenSize(0.dp, 0.dp)) }
+            val eventScope = remember { CoroutineScope(SupervisorJob() + Dispatchers.Main) }
+
+            YourAppTheme {
+                Surface(
+                    color = MaterialTheme.colorScheme.background,
+                    modifier = Modifier
+                        .fillMaxSize()
+                        .onSizeChanged { screenSize = ScreenSize(it.width.dp, it.height.dp) }
+                        .onKeyEvent {
+                            // See back handling section in the docs below!    
+                            onKeyEvent(it, events, eventScope) 
+                        }
+                        .focusRequester(requester)
+                        .focusable()
+                        .onFocusChanged { hasFocus = it.hasFocus }
+                ) {
+                    WebNodeHost(
+                        screenSize = screenSize,
+                        onBackPressedEvents = events.receiveAsFlow(),
+                    ) { 
+                        RootNode(buildContext = it)
+                    }
+                }
+
+                if (!hasFocus) {
+                    LaunchedEffect(Unit) {
+                        requester.requestFocus()
+                    }
+                }
+            }
+        }
+    }
+}
+
+```
+
+## Back handling
+
+On Android back events are handled automatically.
+
+In the above [desktop](#desktop) and [web](#web) examples there are a references to an `onKeyEvent` method.
+
+You can implement as such, and configure the keybinding which `Key` should trigger a back event:
```

Comment:
rephrase
"You can configure any `KeyEvent` to trigger a back event via the events Channel. In this example the OnBackPressed event is launched when the backspace key is pressed down."